express-rest-api/README.md

```diff
@@ -0,0 +1,7 @@
+# Express REST API example
+
+This example demonstrate how you can create test code using Mocha.js
+[Express](https://expressjs.com) used to create a server and [supertest](https://github.com/visionmedia/supertest) used for HTTP assertions, then Mocha validates the result. 
+
+## Commands
+- `npm test` - run the mocha test code 
```

Comment:
It could be confusing to someone why there isnt a Mocha config file. Could you add something like. `"No config file is needed for Mocha as the app uses default values"` pls? I know it might seem obvious but we want these example apps to be very explicit.

Reply:
Yeap good point! Even minor things can be confusing for first-time users.